controllers/spec/template/java-runtime-log4j.yaml.tmpl

```diff
@@ -19,64 +19,62 @@ Configuration:
         eventEol: true
       {{ else }}
       PatternLayout:
-        Pattern: "%d{ISO8601_OFFSET_DATE_TIME_HHMM} [%t] %-5level %logger{36} - %msg%n"
+        Pattern: \"%d{ISO8601_OFFSET_DATE_TIME_HHMM} [%t] %-5level %logger{36} - %msg%n\"
       {{ end }}
 
     RollingRandomAccessFile:
       name: RollingRandomAccessFile
-      fileName: "${sys:pulsar.function.log.dir}/${sys:pulsar.function.log.file}.log"
-      filePattern: "${sys:pulsar.function.log.dir}/${sys:pulsar.function.log.file}.%d{yyyy-MM-dd-hh-mm}-%i.log.gz"
```

Comment:
use `-mm` in file pattern and combine with `TimeBasedTriggeringPolicy` will result in rolling log file in every 1 minute

with `-hh` it will roll in every 1 hour, maybe we should use `yyyy-MM-dd` to roll the file log in every day?

Reply:
+1 to make it to `yyyy-MM-dd`.

Reply:
updated